### PR TITLE
LPS-74544

### PR DESCRIPTION
**Pull request by petershin**
Example: [AnalyticsNoSQLConfiguration.java#L40](https://github.com/liferay/com-liferay-osb-lcs-private/blob/7.0.x-private/osb-lcs-analytics-web/src/main/java/com/liferay/osb/lcs/analytics/web/configuration/AnalyticsNoSQLConfiguration.java#L40)

### CODE REVIEW

**Comment by petershin** [comment]
ci:test

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/JavaLongLinesCheck.java [hunk 1]
```diff
@@ -82,6 +82,12 @@ protected String doProcess(
 					continue;
 				}
 
+				if (StringUtil.startsWith(trimmedLine, StringPool.AT) &&
+					!trimmedLine.matches(".*\\s+.*")) {
```
Comment (hhuijser) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/JavaLongLinesCheck.java]: Why not simply `!trimmedLine.contains(StringPool.SPACE)`

Comment (hhuijser) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/JavaLongLinesCheck.java]: or... if you using `matches` already, merge both and do `trimmedLine.matches("@\\S+")`

Comment (petershin) [reply modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/JavaLongLinesCheck.java]: Updated the pull

**Comment by hhuijser** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/54315

**Pull request closed without merging**